### ISSUE TITLE
Tooltip component needs aria-label even when not using Carbon icon

### ISSUE THREAD

**Issue by rstrangfeld** [IBM/carbon-components-react#2103]
The Tooltip component lets you use either (option 1) a Carbon icon, or (option 2) something else that you supply, as the displayed trigger element. Internally the Tooltip component wraps that element in a div that handles clicks by displaying the tooltip.

If you choose option 1, a carbon icon, the wrapper div gets whatever aria-label the caller specified for the Tooltip component. This behavior was added to the Tooltip to fix https://github.com/IBM/carbon-components-react/issues/1697.

However, if you choose option 2 and supply your own trigger element, then the wrapper div doesn't get any aria-label. It fails the "All interactive elements must have an interactive name" accessibility test, just like option 1 did before 1697.

The fix for 1697 was good, but it only fixed the issue for one of the two cases. Option 2 needs a similar fix.

Note that you can't just put the aria-label on the trigger element you supply... it's the wrapper div created by Tooltip that is the interactive element.

The fix for 1697 was a one-line change. That will probably be the case here too.

**Comment by rstrangfeld** [comment]
It looks like stale bot is threatening to close this issue because of inactivity. We're still interested in a fix. What would need need to happen for that to come about?
Thanks!

**Comment by dakahn** [comment]
@asudoh I see you pushed the changes for [the referenced commit](https://github.com/carbon-design-system/carbon-components-react/commit/9d00fba39a85f9de7319aaef50379b789d973d4c). What do you think of the proposed changes here?

**Comment by asudoh** [comment]
@dakahn It came from hitting "merge" button from a PR from a contributor. Anyhow, I have an impression that the (text-based) content inside the trigger button should be read in scenario. Thoughts?

**Comment by dakahn** [comment]
100% agree
